src/rulesets/LinearAlgebra/blas.jl

```diff
@@ -129,8 +124,9 @@ function rrule(::typeof(gemv), tA::Char, α::T, A::AbstractMatrix{T},
     return y, gemv_pullback
 end
 
-function rrule(::typeof(gemv), tA::Char, A::AbstractMatrix{T},
-               x::AbstractVector{T}) where T<:BlasFloat
+function rrule(
+    ::typeof(gemv), tA::Char, A::AbstractMatrix{T}, x::AbstractVector{T}
+) where T<:BlasFloat
```

Comment:
This is one of the few actual style changes -- it now matches the style we use everywhere else